### ISSUE TITLE
AJAX call on add/edit

### ISSUE THREAD

**Issue by mariashahid** [baig/jquerymobile-editablelistview#13]
Can I add ajax calls to add and delete buttons? If yes, can you please provide an example?

Also, Will this plugin work in mobile only since it uses jquery mobile?

**Comment by baig** [comment]
> Can I add ajax calls to add and delete buttons? If yes, can you please provide an example?

I don't think you can directly attach events to add or delete buttons. But you can listen to the `listviewchange` which is triggered when the editable-listview comes out of the edit mode and then make an Ajax call based on any change in the list contents. See the [Events](https://github.com/baig/jquerymobile-editablelistview/wiki/Events) page for more information.
If you can share the context of your particular use case, then may be I'll be in a better position to suggest a solution.

> Also, Will this plugin work in mobile only since it uses jquery mobile?

jQuery mobile is just a requirement to run this plugin. The plugin does not care if it's being run on a mobile or not. If you design a Tablet or Desktop UI using jQuery mobile, then there's no reason that this plugin should not work.
